LiteFarmOrg/LiteFarm
# LF-2528 temperature sensor preview copy fix

Read the latest comment on [LF-2528](https://lite-farm.atlassian.net/browse/LF-2528).

To Test:
1. Have sensors and sensor readings in the database.
2. Long press a sensor on the map to see the preview.
3. The copy should be "Current soil temp" instead of "Temperature" for English.
